docs/docs-content/byoos/usecases/vmware/konvoy.md

```diff
@@ -0,0 +1,465 @@
+    | `packer.template`            | The name of the RHEL template in vSphere that you created using the ISO file of the RHEL version you want to use.                  |
+    | `packer.ssh_username`        | The username to SSH into the Packer VM. Specify the username you used to log in to the RHEL VM before converting it to a template. |
+    | `packer.ssh_password`        | The password to SSH into the Packer VM. Specify the password you used to log in to the RHEL VM before converting it to a template. |
+
+    <details>
+
+    <summary>Example RHEL configuration file</summary>
+
+    The parameters that need to be replaced are highlighted in the example configuration file below.
+
+    ```yaml {9-16,22,23}
+    ---
+    download_images: true
+    build_name: "rhel-88"
+    packer_builder_type: "vsphere"
+    guestinfo_datasource_slug: "https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo"
+    guestinfo_datasource_ref: "v1.4.0"
+    guestinfo_datasource_script: "{{guestinfo_datasource_slug}}/{{guestinfo_datasource_ref}}/install.sh"
+    packer:
+      cluster: "Cluster2"
+      datacenter: "Datacenter"
+      datastore: "example-datastore"
+      folder: "internal"
+      insecure_connection: "true"
+      network: "DEV-NETWORK"
+      resource_pool: "rp-dev"
+      template: "internal/rhel"
+      vsphere_guest_os_type: "rhel8_64Guest"
+      guest_os_type: "rhel8-64"
+      # goss params
+      distribution: "RHEL"
+      distribution_version: "8.8"
+      ssh_username: "**********"
+      ssh_password: "*********"
+      linked_clone: false
+      ansible_sudo_pass: "********"
+      use_sudo: false
+      NOPASSWD: ALL
+    # ssh_private_key_file = "" # can be exported as environment variable 'SSH_PRIVATE_KEY_FILE'
+    # ssh_agent_auth: false  # is set to true, ssh_password and ssh_private_key will be ignored
+    ```
+
+    </details>
+
+7.  Review the **images/common.yaml** file and modify the file if necessary. The **common.yaml** file is where you
+    specify the Kubernetes version. Refer to the[Konvoy](../../../integrations/konvoy.md) pack reference page to learn
+    about supported Konvoy versions.
+
+    ```yaml hideClipboard {2}
+    ---
+    kubernetes_version: "1.27.6"
+    download_images: true
+    packer:
+      goss_arch: amd64
+      goss_entry_file: goss/goss.yaml
+      goss_format: json
+      goss_format_options: pretty
+      goss_inspect_mode: false
+      goss_tests_dir: goss
+      goss_url:
+      goss_vars_file: ansible/group_vars/all/system.yaml
+      goss_version: 0.3.16
+    ```
+
+8.  Update **ansible/roles/fips/tasks/redhat-8.yaml** to address potential timeout issues. Add the following lines to
+    the `ansible.builtin.reboot` task.
+
+        ```yaml
+        ansible.builtin.reboot:
+          post_reboot_delay: 120
+          pre_reboot_delay: 30
+          connect_timeout: 120
+        ```
+
+    {/* prettier-ignore */}
```

Comment:
What does this do?

Reply:
Oh man, @addetz prettier kept refusing to let the current indentation work, which is the only way to achieve the correct indentation. Prettier kept pushing the `<summary> element to the far left and breaking the build. The comment above tells prettier to backoff.